pkg/utils/k8sutil.go

```diff
@@ -453,21 +453,33 @@ func CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic,
 		Do().Into(&f)
 	if err != nil {
 		if k8sErrors.IsNotFound(err) {
+			funcLabels := map[string]string{}
+			for _, label := range labels {
+				k, v := getKV(label)
```

Comment:
Yeah so I should not support the environment values at this moment. Will add it later when we need it.